### ISSUE TITLE
Missing attr value quotes

### ISSUE THREAD

**Issue by svinkle** [komplexb/komplexb.github.io#10]
Minor thing but missing quotes in `aria-hidden=true` here. Also, remove the empty `class`. 

https://github.com/komplexb/komplexb.github.io/blob/2bd4fd33944c89e3264b827a6c24061602624cc5/index.html#L141
